ci-operator/config/shiftstack/machine-api-provider-openstack/shiftstack-machine-api-provider-openstack-master.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.9
+binary_build_commands: make build
+build_root:
+  from_repository: true
+canonical_go_repository: sigs.k8s.io/machine-api-provider-openstack
```

Comment:
Should this be prepended by `sigs.k8s.io/`? I think its supposed to just be: `shiftstack/machine-api-provider-openstack` as it is right now, but will ideally be `github.com/openshift/machine-api-provider-openstack` once we move it into the openshift org.